hardware/README.md

```diff
@@ -74,20 +75,22 @@ If the configuration step was already executed, you may just call:
 ```bash
     make image
 ```
+**Note** that you must still build the software tools on the POWER target system.
 
 A simulation model (for the simulator defined by the environment variable `$SIMULATOR`) may be created
-via the target model:
+via the target `model`:
 
 ```bash
     make config model
 ```
+This will also build the software tools and the PSLSE which are required to run the simulation.
 
-Please refer to `$SNAP_ROOT/hardware/Makefile` for more supported targets like clean, gitclean, create_environment, copy, ...
+Please refer to `$SNAP_ROOT/hardware/Makefile` for more supported targets like clean, gitclean, create_environment, ...
 
 # Action wrapper
 
-The path to the set of actions that shall be included is defined via the environment variable `$ACTION_ROOT`.
-**Currently it has to point to a directory within** [./action_examples](./action_examples).
+The environment variable `$ACTION_ROOT` defines the path to the set of actions that shall be included.
+**Currently it has to point to a directory within** [./action_examples](./action_examples). 
```

Comment:
Please remove trailing whitespace (since an empty line is following, it doesn't look like the goal is a line break at this point)